Source/Controllers/SPAppController.m

```diff
@@ -195,10 +195,31 @@ - (void)awakeFromNib
  * Initialisation stuff after launch is complete
  */
 - (void)applicationDidFinishLaunching:(NSNotification *)notification {
+
+    // Send time interval for non-critical logs
+    // must set before calling AppCenter.start
+    // 5 mins?
+    [MSACAnalytics setTransmissionInterval:60*5];
+
+    //You must set the delegate before calling AppCenter.start
+    [MSACCrashes setDelegate:self];
```

Comment:
Why do we need that - just for SPLogs? I would leave it disabled and without the delegate methods for now - it needs more work and love... :)